wikipedia-dark.user.css

```diff
@@ -110,6 +110,52 @@
   .mw-echo-ui-toggleReadCircleButtonWidget:hover .mw-echo-ui-toggleReadCircleButtonWidget-circle-unread {
     background-color: darkslategray !important;
   }
+  .mw-echo-ui-menuItemWidget > .oo-ui-labelElement-label {
+    color: #919191;
+  }
+  .mw-collapsible:not([class*="navbox-"]):not([class*="wpb"]) th {
+    background-color: #333 !important;
+  }
+  .mw-collapsible:not([class*="navbox-"]) td {
+    background-color: #222 !important;
+  }
+  .wikipediauserbox td {
+    color: #9a9a9a !important;
+  }
+  .oo-ui-textInputWidget .oo-ui-inputWidget-input {
+    color: #9a9a9a;
+  }
+  .oo-ui-toolbar-bar, .oo-ui-window-head, .oo-ui-window-foot,
+  .oo-ui-popupToolGroup-tools {
+    background: #222 !important;
+  }
+  table.diff, td.diff-otitle, td.diff-ntitle {
+    background-color: #333;
+  }
+  .oo-ui-tool.oo-ui-widget-enabled > .oo-ui-tool-link:hover,
+  .oo-ui-tool.oo-ui-widget-enabled > .oo-ui-tool-link:active,
+  .oo-ui-tool.oo-ui-widget-enabled > .oo-ui-tool-link:active:focus,
+  .oo-ui-tool.oo-ui-widget-enabled.oo-ui-tool-active > .oo-ui-tool-link,
+  .oo-ui-tool.oo-ui-widget-enabled.oo-ui-popupToolGroup-active > .oo-ui-tool-link {
+    background-color: #333;
```

Comment:
Merge table.diff, td.diff-otitle, td.diff-ntitle with the rule below.

Reply:
Two main thoughts here:

  - I'm trying to avoid combining rules with and without `!important`, since in some cases adding an unnecessary `!important` will cause unintentional style breakages elsewhere. There are several cases like that right now, and it's likely to grow in the future. If we can redo those general selectors so we don't need `!important` sprinkled all over the place to override them, more of these could be safely combined.
  - I'm also trying not to combine completely unrelated elements just because they happen to have the same rules at the moment (ex: a `button` border and the color of a `p` element). This can come back to bite us in the future when we need to change just one of them, and we're not sure if that selector was grouped with the others intentionally, or just because. I'll start combining the ones with similar roles/types/purposes, but there should be some separation. Grouping them together can allow for easier full-stylesheet changes if we want to change a particular color globally (or variables, but that's later).

Reply:
Ok, most of the identical selectors have been combined, where it makes sense to do so.